### DISCUSSION TITLE
Help to setup

### DISCUSSION THREAD

**Discussion by omerstudio** [CosmosOS/Cosmos#2605] [Q&A]
i got a project tomorrow and i just wanna load my operating system on VMware and it gets stuck on 'Waiting for debug connection' but when i run it from Visual Studio, there is no problem. Anyone could help? Im writing this in 01:05 (1:05 A.M. Turkey)

i installed the .vmdk filesystem too but it gets stuck.

**Comment by ascpixi** [comment]
If you're booting your system from the ISO created by Cosmos after launching your project, you'll need to use the ISO image build profile instead of the VMWare one. The VMWare build profile generates ISO files that expect a remote debugger to be running on the host machine.

**Comment by randomusert** [comment]
it waits to a remote debugger to be started. you have to change the properties on your project on debug section.
